--- a/config.c
+++ b/config.c
@@ -30,8 +30,8 @@
 #include "xmalloc.h"
 #include "xsnprintf.h"
 
-static const uint32_t default_foreground = 0xffffff;
-static const uint32_t default_background = 0x242424;
+static const uint32_t default_foreground = 0xdcdccc;
+static const uint32_t default_background = 0x111111;
 
 static const size_t min_csd_border_width = 5;
 
@@ -48,23 +48,23 @@
 
 static const uint32_t default_color_table[256] = {
     // Regular
-    0x242424,
-    0xf62b5a,
-    0x47b413,
-    0xe3c401,
-    0x24acd4,
-    0xf2affd,
-    0x13c299,
-    0xe6e6e6,
+    0x222222,
+    0xcc9393,
+    0x7f9f7f,
+    0xd0bf8f,
+    0x6ca0a3,
+    0xdc8cc3,
+    0x93e0e3,
+    0xdcdccc,
 
     // Bright
-    0x616161,
-    0xff4d51,
-    0x35d450,
-    0xe9e836,
-    0x5dc5f8,
-    0xfeabf2,
-    0x24dfc4,
+    0x666666,
+    0xdca3a3,
+    0xbfebbf,
+    0xf0dfaf,
+    0x8cd0d3,
+    0xfcace3,
+    0xb3ffff,
     0xffffff,
 
     // 6x6x6 RGB cube
@@ -2994,7 +2994,7 @@
         .use_custom_underline_offset = false,
         .box_drawings_uses_font_glyphs = false,
         .underline_thickness = {.pt = 0., .px = -1},
-        .dpi_aware = false,
+        .dpi_aware = true,
         .bell = {
             .urgent = false,
             .notify = false,
